zerver/migrations/0377_message_edit_history_format.py

```diff
@@ -0,0 +1,165 @@
+import time
+from typing import List, Optional
+
+import orjson
+from django.db import migrations, transaction
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import Min, Model
+from typing_extensions import TypedDict
+
+BATCH_SIZE = 1000
```

Comment:
I think this should probably be 10K, given that messages with non-null edit history are likely sparse.

Reply:
Changed this.